compute/admin_guide/install/install_defender/install_app_embedded_defender_fargate.adoc

```diff
@@ -183,51 +183,120 @@ Prisma Cloud supports the following task definition formats:
 }
 ----
 
-*Example of the equivalent task definition as a CloudFormation template:*
+*Example of the equivalent task definition as a CloudFormation template in JSON:*
 
 ----
 {
-  "Type" : "AWS::ECS::TaskDefinition",
-  "Properties": {
-      "ContainerDefinitions": [
-        {
-          "Name": "web",
-          "Image": "nginx",
-          "EntryPoint": [
+  "AWSTemplateFormatVersion": "2010-09-09",
+  "Resources": {
+    "fargateTaskDefinition": {
+      "Type": "AWS::ECS::TaskDefinition",
+      "Properties": {
+        "ExecutionRoleArn": "arn:aws:iam::1234567891234:role/ecsTaskExecutionRole",
+        "ContainerDefinitions": [
+          {
+            "Name": "test-server",
+            "Image": "1234567891234.dkr.ecr.us-east-1.amazonaws.com/user:ubuntu-test-server",
+            "MemoryReservation": "",
+            "Essential": true,
+            "PortMappings": [],
+            "Cpu": 256,
+            "Memory": 512,
+            "EntryPoint": [
               "/http_server"
+            ],
+            "EnvironmentFiles": [],
+            "LogConfiguration": {
+              "LogDriver": "awslogs",
+              "Options": {
+                "awslogs-group": "/ecs/user-tracer-test",
+                "awslogs-region": "us-east-1",
+                "awslogs-stream-prefix": "ecs"
+              }
+            }
+          }
+        ],
+        "Memory": "512",
+        "TaskRoleArn": "arn:aws:iam::1234567891234:role/ecsTaskExecutionRole",
+        "Family": "TASK-NAME",
+        "RequiresCompatibilities": [
+          "FARGATE"
+        ],
+        "NetworkMode": "awsvpc",
+        "Cpu": "256",
+        "InferenceAccelerators": [],
+        "Volumes": [],
+        "Tags": []
+      }
+    },
+    "HelloLambdaRole": {
+      "Type": "AWS::IAM::Role",
+      "Properties": {
+        "RoleName": "HelloLambdaRole1",
+        "AssumeRolePolicyDocument": {
+          "Statement": [
+            {
+              "Effect": "Allow",
+              "Principal": {
+                "Service": "lambda.amazonaws.com"
+              },
+              "Action": "sts:AssumeRole"
+            }
           ]
         }
-      ],
-      "Cpu" : 256,
-      "ExecutionRoleArn": "arn:aws:iam::112233445566:role/ecsTaskExecutionRole",
-      "Family": "webserver",
-      "Memory" : 512,
-      "NetworkMode" : "awsvpc",
-      "RequiresCompatibilities" : [
-          "FARGATE"
-      ]
+      }
     }
+  }
 }
 ----
-
-[NOTE]
```

Comment:
@ssugandh even though we removed this NOTE as I suggested, I do think we should mention somewhere that the user can use either a template with just the task definition or a full CloudFomration template. Maybe it can fit under the "Supported task definition formats" section, under the bullet talking about CloudFormation

Reply:
This is mentioned at line no 160. Please see